msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase850455.java

```diff
@@ -114,7 +114,7 @@ public String getTempUserType() {
 
     @Override
     public String getAuthority() {
-        return "https://login.microsoftonline.de/common";
+        return "https://login.microsoftonline.us/common";
```

Comment:
Is it possible to be getting this from the query we're running, rather than code the value?  Seems like it would be safer.